src/main/webapp/testScript.js

```diff
@@ -0,0 +1,22 @@
+//button styling 
+mdc.ripple.MDCRipple.attachTo(document.querySelector('.foo-button'));
+
+//this funciton will implement the fetch to grab the list of comments from the database and call to the newDiv()
+//function to place the comments into the feed div.
+window.onload = function display_feed(){
+    
+}
+
+//creates a new div for each new comment
+function newDiv(){
+    //make this list by calling a fetch function 
+    var test_comment = ["Hello mr.rep", "hi mr.townsman", "hi doctor"];
```

Comment:
consider making it just take in a list of comments as an argument for now and then we can add a function that queries for the comments from datastore later

Reply:
Updated in the final version of the function